src/transformers/models/oneformer/modeling_oneformer.py

```diff
@@ -1633,11 +1633,13 @@ def forward(
 
 
 class OneFormerTransformerDecoderSelfAttentionLayer(nn.Module):
-    def __init__(self, embed_dim, num_heads, dropout=0.0, activation="relu", normalize_before=False):
+    def __init__(
+        self, embed_dim, num_heads, dropout=0.0, activation="relu", normalize_before=False, layer_norm_eps=1e-05
```

Comment:
We don't have `config` argument here, so I need to add `layer_norm_eps`. The `config.layer_norm_eps` will be passed from the module(s) that contain(s) this module.